### PR TITLE
Cor-179: Fix individual relationship bug

### PR DESCRIPTION
**Pull request by senyaoh**
- Display individual's name in relationship taken from first name and last name input field before it's saved
- Fix saved individual's relationship link

### CODE REVIEW

File: api/pkg/apps/webapp/templates/individual.gohtml [hunk 1]
```diff
@@ -564,14 +564,22 @@
           relationships.forEach((relationship, idx) => {
             const li = document.createElement('li');
             li.className = 'list-group-item';
-            const firstPartyName = parties.find(({ id }) => id === relationship.firstParty)?.name;
+            let firstPartyName
+            if (window.location.pathname == '/individuals/new') {
+              const lastName = document.getElementsByName('attribute[21079bbc-e04b-4fe8-897f-644d73af0d9e]')[0].value
+              const firstName = document.getElementsByName('attribute[8514da51-aad5-4fb4-a797-8bcc0c969b27]')[0].value
```
Comment (nilueps) [inline api/pkg/apps/webapp/templates/individual.gohtml:570]:
```suggestion
              const lastName = document.querySelector('[value={{.Individual.Attributes.Get "firstName"}}]').value
              const firstName = document.getElementsByName('attribute[8514da51-aad5-4fb4-a797-8bcc0c969b27]')[0].value
```

Comment (senyaoh) [reply api/pkg/apps/webapp/templates/individual.gohtml:570]: Hey @nilueps, the code you provided doesn't seem to work, I got a `Failed to execute 'querySelector' on 'Document': '[value=]' is not a valid selector.` error

File: api/pkg/apps/webapp/templates/individual.gohtml [hunk 2]
```diff
@@ -564,14 +564,22 @@
           relationships.forEach((relationship, idx) => {
             const li = document.createElement('li');
             li.className = 'list-group-item';
-            const firstPartyName = parties.find(({ id }) => id === relationship.firstParty)?.name;
+            let firstPartyName
+            if (window.location.pathname == '/individuals/new') {
```
Comment (nilueps) [inline api/pkg/apps/webapp/templates/individual.gohtml:568]: Javascript is weird, privilege strict equal as much as possible.
```suggestion
            if (window.location.pathname === '/individuals/new') {
```

File: api/pkg/apps/webapp/templates/individual.gohtml (hunk 1 above)
Comment (nilueps) [reply api/pkg/apps/webapp/templates/individual.gohtml:570]: Hey sorry, I didn't know that comment already got submitted while I was writing it! Check latest edit

**Review by nilueps — APPROVED** [review]

**Pull request merged**